src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md

```diff
@@ -25,7 +25,7 @@ This cmdlet takes a **VirtualMachineScaleSet** object as input.
 
 ### Example 1: Create a VMSS
 ```
-PS C:\># Common
+PS C:\> # Common
```

Comment:
Hi Mark, I'm used to doing multiple PRs when I'm actively working on Docs, so I normally do a find and replace on the prompt spacing then do a PR. I usually do multiple PRs to avoid merge conflicts. The fix to New-AzureRmVmss.md will be fixed in my next PR which will be later today.